docs/iot-gateway/config/mqtt.md

```diff
@@ -11,585 +11,940 @@ redirect_from:
 
-This configuration section is optional.
-Configuration, provided in this section will be used to get information from the broker about disconnecting device.
-If your device just disconnects from MQTT broker and waits for commands/updates, you need to send a message to the Gateway and inform it that device is disconnected from the broker.
 
-Select basic or advanced MQTT configuration:
+Example of the attribute requests configuration:
 
-{% capture mqttdisconnectrequestsubsection %}
-Basic<small></small>%,%basic%,%templates/iot-gateway/mqtt-connector/disconnect-request-subsection-basic.md%br%
-Advanced<small></small>%,%advanced%,%templates/iot-gateway/mqtt-connector/disconnect-request-subsection-advanced.md{% endcapture %}
+```json
+"attributeUpdates": [
+  {
+    "retain": true,
+    "deviceNameFilter": ".*",
+    "attributeFilter": "firmwareVersion",
+    "topicExpression": "sensor/${deviceName}/${attributeKey}",
+    "valueExpression": "{\"${attributeKey}\":\"${attributeValue}\"}"
+  }
+],
+```
+{: .copy-code}
+
+#### Device RPC methods
 
-{% include content-toggle.liquid content-toggle-id="mqttdisconnectrequestsubsection" toggle-spec=mqttdisconnectrequestsubsection %}
+Device RPC (Remote Procedure Call) methods allow you to send commands from ThingsBoard to your devices through the MQTT connector. The gateway acts as an intermediary, translating ThingsBoard RPC calls into MQTT messages that your devices can understand.
 
-**Now let's review an example.**
+There are two types of RPC requests supported by the MQTT connector:
 
-Use a terminal to simulate sending a message from the device to MQTT broker:
+1. **Two-way (with response)** - The gateway sends a request to the device and waits for a response. This is useful when you need to get data back from the device.
+2. **One-way (without response)** - The gateway sends a request to the device without expecting a response. This is useful for simple commands that don't require confirmation.
 
-```bash
-mosquitto_pub -h 127.0.0.1 -p 1883 -t "sensor/disconnect" -m '{"serialNumber": "SN-001"}'
+
+| **Parameter**                                                                         | **Description**                                                                                                                                                                                 |
+|:--------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| requestsMapping.serverSideRpc[].deviceNameFilter                                      | Regular expression device name filter, is used to determine which devices this RPC configuration applies to.See [expression](#expression-types)                                                 |
```

Comment:
applies to.See [expression](#expression-types) 


Use the same formatting for all descriptions.